src/wallet/rpcdump.cpp

```diff
@@ -575,15 +578,33 @@ UniValue importwallet(const JSONRPCRequest& request)
         int64_t nFilesize = std::max((int64_t)1, (int64_t)file.tellg());
         file.seekg(0, file.beg);
 
+        static const std::string HEADER_STRING = "# Wallet dump created by Bitcoin";
```

Comment:
nit: Should use `PACKAGE_NAME` instead of "Bitcoin"?

Reply:
`PACKAGE_NAME` is "Bitcoin Core[default wallet]", but `dumpwallet` uses `"Bitcoin %s" CLIENT_BUILD`. I assume you can use importwallet on wallets dumped using an earlier build right?

Reply:
Oh sure that should work. I see it is only a log-print and not a error, so should be fine to keep this as is.